src/lib/css-tag.ts

```diff
@@ -60,9 +60,14 @@ export const unsafeCSS = (value: unknown) => {
   return new CSSResult(String(value), constructionToken);
 };
 
-const textFromCSSResult = (value: CSSResult) => {
+const textFromCSSResult = (value: CSSResult | number) => {
   if (value instanceof CSSResult) {
     return value.cssText;
+  } else if (typeof value === 'number' && isFinite(value) && value === value) {
```

Comment:
Since there's no security risk from Infinity or NaN, I think we should allow them rather than throw. They might not result in valid CSS, but at least we wouldn't throw and abort rendering. It's also less code and checks.

Reply:
updated 🤗